fast_and_reliable.py

```diff
@@ -0,0 +1,37 @@
+import math
+import time
+
+
+def average_runtime(words):
+    timeFirst = time.time()
+    for i in range(0, 1000):
+        if "zwitterion" in words:
+             continue
+    timeSecond = time.time()
+    return (timeSecond - timeFirst) / 1000
+
+
+word_list = []
```

Comment:
Dont write code outside a function in python - it will be executed when you dont want it to execute - when you are importing from this file, for example